enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/InternalFlatFileRealmTest.java

```diff
@@ -131,6 +143,174 @@ public void shouldOnlyReloadUsersOrRolesIfNeeded() throws Throwable
         assertSetUsersAndRolesNTimes( true, true, 1, 1 );
     }
 
+    @Test
+    public void shouldAssignAdminRoleToDefaultUser() throws Throwable
+    {
+        // Given
+        InternalFlatFileRealm realm = internalTestRealmWithUsers( Collections.emptyList(), Collections.emptyList() );
+
+        // When
+        realm.initialize();
+        realm.start();
+
+        // Then
+        assertThat( realm.getUsernamesForRole( PredefinedRoles.ADMIN ),
+                contains( InternalFlatFileRealm.INITIAL_USER_NAME ) );
+    }
+
+    @Test
+    public void shouldAssignAdminRoleToSpecifiedUser() throws Throwable
+    {
+        // Given
+        InternalFlatFileRealm realm = internalTestRealmWithUsers( Arrays.asList( "neo4j", "morpheus", "trinity" ),
+                Collections.singletonList( "morpheus" ) );
+
+        // When
+        realm.initialize();
+        realm.start();
+
+        // Then
+        assertThat( realm.getUsernamesForRole( PredefinedRoles.ADMIN ), contains( "morpheus" ) );
+        assertThat( realm.getUsernamesForRole( PredefinedRoles.ADMIN ).size(), equalTo( 1 ) );
+    }
+
+    @Test
+    public void shouldAssignAdminRoleToOnlyUser() throws Throwable
+    {
+        // Given
+        InternalFlatFileRealm realm =
+                internalTestRealmWithUsers( Collections.singletonList( "morpheus" ), Collections.emptyList() );
+
+        // When
+        realm.initialize();
+        realm.start();
+
+        // Then
+        assertThat( realm.getUsernamesForRole( PredefinedRoles.ADMIN ), contains( "morpheus" ) );
+        assertThat( realm.getUsernamesForRole( PredefinedRoles.ADMIN ).size(), equalTo( 1 ) );
+    }
+
+    @Test
+    public void shouldNotAssignAdminToNonExistentUser() throws Throwable
+    {
+        // Given
+        InternalFlatFileRealm realm = internalTestRealmWithUsers( Collections.singletonList( "neo4j" ),
+                Collections.singletonList( "morpheus" ) );
+
+        // Expect
+        exception.expect( InvalidArgumentsException.class );
+        exception.expectMessage( "No roles defined, and default admin user 'morpheus' does not exist. Please use `" +
+                SetDefaultAdminCommand.COMMAND_NAME + "` to select a valid admin." );
```

Comment:
Now it says to use `set-default-admin`. Maybe we should also mention that this command lives in `neo4j-admin`?